consensus/istanbul/backend/handler.go

```diff
@@ -133,6 +133,11 @@ func (sb *Backend) handleConsensusMsg(peer consensus.Peer, payload []byte) error
 			return nil
 		}
 
+		if _, err := sb.valEnodeTable.GetAddressFromNodeID(peer.Node().ID()); err != nil {
```

Comment:
This is a great first pass.

However, the valEnodeTable will contain entries for all registered and elected validators.  And the contents of this table is not the most suitable for the query "get current elected validators".  Specifically this table's purpose to do map a validator signing address to that validator's enode url (akin to a DNS table).

The best way to get the current elected validators is via the blockchain header's.  You may already know this, but at each epoch's last block, there will be an elected validator set diff (with the initial set stored in the genesis block).  So to get the current elected validators, you would need to retrieve those blocks, and calculate the current set.  This function does that:  https://github.com/celo-org/celo-blockchain/blob/master/consensus/istanbul/backend/backend.go#L275